StratisCore.UI/src/app/wallet/block-explorer/block-explorer.component.html

```diff
@@ -0,0 +1,31 @@
+<div *ngIf="(blockExplorerService.blocks | async) let _blocks;"
```

Comment:
General comment on block explorer:
Imo a block explorer should never be in the wallet, nothing contacting external APIs should be inside the wallet. 
A wallet is a wallet and should never be more, everything added on top adds security risks. 
I know it's not important for 2.0.0.0 and I'm ok to push this PR in with it, but it should be removed imo. 
To be discussed in a future standup.